lib/Sema/CSDiagnostics.cpp

```diff
@@ -6224,3 +6224,15 @@ bool CoercionAsForceCastFailure::diagnoseAsError() {
       .highlight(getSourceRange());
   return true;
 }
+
+bool KeyPathRootTypeMismatchFailure::diagnoseAsError() {
```

Comment:
Perhaps this method can be `const` as well?

Reply:
This is an override of `ContextualFailure` so I think is not possible make it const :)

Reply:
Aha, makes sense.